Timer.dcm

```diff
@@ -120,8 +120,32 @@ K timer
 F https://www.onsemi.com/pub/Collateral/MC1455-D.PDF
 $ENDCMP
 #
-$CMP MCP7940N
-D Real-Time Clock, I2C, Battery Backup
+$CMP MCP7940N-MNY
```

Comment:
Should say `MCP7940n-xMNY`.  Right now the symbol editor gives an error about this.